lib/data/data_source/tour_info_data_source_impl.dart

```diff
@@ -49,7 +49,7 @@ class TourInfoDataSourceImpl implements TourInfoDataSource {
 
   // 행사정보조회
   @override
-  Future<List<TourDto>> getSearchFestival({required String eventStartDate, int pageNo = 1}) async {
+  Future<List<TourDto>> getSearchFestival({required String eventStartDate, required String eventEndDate, int pageNo = 1}) async {
     const String apiName = 'searchFestival1';
     final Response response;
     response = await _dio.get('$baseUrl/$apiName?pageNo=$pageNo&MobileOS=$_mobileOs&MobileApp=MobileApp&_type=json&arrange=Q&eventStartDate=$eventStartDate&serviceKey=$key');
```

Comment:
dataSourceImpl에 추가된 eventEndDate를 활용하시려면 API URL에도 포함되어야 할 것 같습니다..!
```suggestion
  Future<List<TourDto>> getSearchFestival({required String eventStartDate, required String eventEndDate, int pageNo = 1}) async {
    const String apiName = 'searchFestival1';
    final Response response;
    response = await _dio.get('$baseUrl/$apiName?pageNo=$pageNo&MobileOS=$_mobileOs&MobileApp=MobileApp&_type=json&arrange=Q&eventStartDate=$eventStartDate&eventEndDate=$eventEndDate&serviceKey=$key');
```

Reply:
꼼꼼하십니다!!
커밋 전 체크 하는 습관을 길러야겠습니다.. ㅜㅜ 감사합니다!!